docs/best-practices.md

```diff
@@ -864,6 +866,29 @@ pvcCleaner:
       kubernetes.io/os: linux
 ```
 
+You can also specify a global nodeSelector option that will be used for all components above except for `otellogs`, `kube-prometheus-stack`,
```

Comment:
Why is `otellogs` excluded here?

Reply:
Grep did not find the key for me, and I saw that in readme we had only `otellogs.daemonset` so I thought we just copy it.
I looked deeper into it and `nodeSelector` is copied a bit differently there:

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/c1964af9c571c35e6a025c6aa8417fb5f2be2433/deploy/helm/sumologic/templates/logs/collector/otelcol/daemonset.yaml#L52

I'm not sure what it means, could you explain?